meson.build

```diff
@@ -0,0 +1,191 @@
+project('rdkafka', 'c', 'cpp', version: '1.4.0')
+cc = meson.get_compiler('c')
+cpp = meson.get_compiler('cpp')
+pkgconfig = import('pkgconfig')
+
+hdr = files(
+	'src/rdkafka.h',
+	'src-cpp/rdkafkacpp.h',
+)
+subdir('meson-include/librdkafka') # hack for header include paths
```

Comment:
```
../../../../tmp/tmpfb3nl0ty/src/librdkafka-1.4.0/meson.build:10:0: ERROR: The "meson-" prefix is reserved and cannot be used for top-level subdir().
```

Reply:
Hm. This isn't actually a problem when the `meson.build` is used through the wrap since it won't be top-level. Should I report that as a meson bug?